runtime/compiler/p/runtime/J9PPCArrayTranslate.spp

```diff
@@ -1313,973 +1264,9 @@ __arrayTranslateTROT:
-	.align	5
-L.vector011011:
-	macroArrayTranslate(`0',`1',`1',`0',`1',`1')
-	.align	5
-L.vector011100:
-	macroArrayTranslate(`0',`1',`1',`1',`0',`0')
-	.align	5
-L.vector011101:
-	macroArrayTranslate(`0',`1',`1',`1',`0',`1')
-	.align	5
-L.vector011110:
-	macroArrayTranslate(`0',`1',`1',`1',`1',`0')
-	.align	5
-L.vector011111:
-	macroArrayTranslate(`0',`1',`1',`1',`1',`1')
-!--
-	.align	5
-L.vector100000:
-	macroArrayTranslate(`1',`0',`0',`0',`0',`0')
-	.align	5
-L.vector100001:
-	macroArrayTranslate(`1',`0',`0',`0',`0',`1')
-	.align	5
-L.vector100010:
-	macroArrayTranslate(`1',`0',`0',`0',`1',`0')
-	.align	5
-L.vector100011:
-	macroArrayTranslate(`1',`0',`0',`0',`1',`1')
-	.align	5
-L.vector100100:
-	macroArrayTranslate(`1',`0',`0',`1',`0',`0')
-	.align	5
-L.vector100101:
-	macroArrayTranslate(`1',`0',`0',`1',`0',`1')
-	.align	5
-L.vector100110:
-	macroArrayTranslate(`1',`0',`0',`1',`1',`0')
-	.align	5
-L.vector100111:
-	macroArrayTranslate(`1',`0',`0',`1',`1',`1')
-	.align	5
-L.vector101000:
-	macroArrayTranslate(`1',`0',`1',`0',`0',`0')
-	.align	5
-L.vector101001:
-	macroArrayTranslate(`1',`0',`1',`0',`0',`1')
-	.align	5
-L.vector101010:
-	macroArrayTranslate(`1',`0',`1',`0',`1',`0')
-	.align	5
-L.vector101011:
-	macroArrayTranslate(`1',`0',`1',`0',`1',`1')
-	.align	5
-L.vector101100:
-	macroArrayTranslate(`1',`0',`1',`1',`0',`0')
-	.align	5
-L.vector101101:
-	macroArrayTranslate(`1',`0',`1',`1',`0',`1')
-	.align	5
-L.vector101110:
-	macroArrayTranslate(`1',`0',`1',`1',`1',`0')
-	.align	5
-L.vector101111:
-	macroArrayTranslate(`1',`0',`1',`1',`1',`1')
-	.align	5
-L.vector110000:
-	macroArrayTranslate(`1',`1',`0',`0',`0',`0')
-	.align	5
-L.vector110001:
-	macroArrayTranslate(`1',`1',`0',`0',`0',`1')
-	.align	5
-L.vector110010:
-	macroArrayTranslate(`1',`1',`0',`0',`1',`0')
-	.align	5
-L.vector110011:
-	macroArrayTranslate(`1',`1',`0',`0',`1',`1')
-	.align	5
-L.vector110100:
-	macroArrayTranslate(`1',`1',`0',`1',`0',`0')
-	.align	5
-L.vector110101:
-	macroArrayTranslate(`1',`1',`0',`1',`0',`1')
-	.align	5
-L.vector110110:
-	macroArrayTranslate(`1',`1',`0',`1',`1',`0')
-	.align	5
-L.vector110111:
-	macroArrayTranslate(`1',`1',`0',`1',`1',`1')
-	.align	5
-L.vector111000:
-	macroArrayTranslate(`1',`1',`1',`0',`0',`0')
-	.align	5
-L.vector111001:
-	macroArrayTranslate(`1',`1',`1',`0',`0',`1')
-	.align	5
-L.vector111010:
-	macroArrayTranslate(`1',`1',`1',`0',`1',`0')
-	.align	5
-L.vector111011:
-	macroArrayTranslate(`1',`1',`1',`0',`1',`1')
-	.align	5
-L.vector111100:
-	macroArrayTranslate(`1',`1',`1',`1',`0',`0')
-	.align	5
-L.vector111101:
-	macroArrayTranslate(`1',`1',`1',`1',`0',`1')
-	.align	5
-L.vector111110:
-	macroArrayTranslate(`1',`1',`1',`1',`1',`0')
-	.align	5
-L.vector111111:
-	macroArrayTranslate(`1',`1',`1',`1',`1',`1')
-
-!----------------------------------------------------------------------
-! endof Handlers
-!----------------------------------------------------------------------
-    	endproc.__arrayTranslateTRTOSimpleVMX:
```

Comment:
Just wanted to highlight that this line does need to be removed even if you aren't removing the above macros. This line should actually be with that rest of the `arrayTranslateTRTOSimpleVMX` but probably got moved apart due to a merge error and was never caught because it was behind an `#if 0`.